mglutil/gui/BasicWidgets/Tk/Dial.py

```diff
@@ -84,7 +84,7 @@ def __init__(self, master=None, type='float',
                  lockPrecision=0, lockShowLabel=0, lockValue=0,
                  lockType=0, lockContinuous=0, lockOneTurn=0, **kw):
 
-	self.master = master = tkinter.Frame(master)
+    self.master = master = tkinter.Frame(master)
```

Comment:
Are all those files indented with 3 spaces? o0

Reply:
I don't remember having manipulated this file, probably this is an unintentional mistake.

Reply:
I've checked the 3spaces indentation was an unintentional/unnoticed change. However, they are related to the AutodockTolls GUI. I recommend discarding the changes